### PR TITLE
add fstype to pv create request

### PR DESCRIPTION
**Pull request by meldafrawi**
longhorn/longhorn#994

Signed-off-by: Mohamed Eldafrawi <mohamed.eldafrawi@rancher.com>

### CODE REVIEW

File: api/model.go [hunk 1]
```diff
@@ -157,7 +157,8 @@ type UpdateReplicaCountInput struct {
 }
 
 type PVCreateInput struct {
-	PVName string `json:"pvName"`
+	PVName   string `json:"pvName"`
+	PVFsType string `json:"fsType"`
```
Comment (yasker) [inline api/model.go]: Use `FSType` as the field name.  It needs to be consistent with the json name.

File: api/volume.go [hunk 2]
```diff
@@ -388,8 +388,13 @@ func (s *Server) PVCreate(rw http.ResponseWriter, req *http.Request) error {
 		return fmt.Errorf("cannot create PV for standby volume %v", vol.Name)
 	}
 
+	// if PV fstype is not defined, default to ext4
+	if input.PVFsType == "" {
```
Comment (yasker) [inline api/volume.go]: Let the manager decide the default FSType.

File: api/model.go (hunk 1 above)
Comment (meldafrawi) [reply api/model.go]: fixed

File: api/volume.go (hunk 2 above)
Comment (meldafrawi) [reply api/volume.go]: moved `fsType` check to manager part.

**Comment by yasker** [comment]
LGTM. Wait for the integration test.

**Pull request merged**